python-test-samples/lambda-mock/README.md

```diff
@@ -0,0 +1,141 @@
+[![python: 3.9](https://img.shields.io/badge/Python-3.9-green)](https://img.shields.io/badge/Python-3.9-green)
+[![AWS: DynamoDB](https://img.shields.io/badge/AWS-DynamoDB-blueviolet)](https://img.shields.io/badge/AWS-DynamoDB-blueviolet)
+[![AWS: S#](https://img.shields.io/badge/AWS-S3-green)](https://img.shields.io/badge/AWS-AWS-S3-green)
+[![test: unit](https://img.shields.io/badge/Test-Unit-blue)](https://img.shields.io/badge/Test-Unit-blue)
+
+# Test AWS Lambda function handlers locally using mocks (Python)
+
+## Introduction
+
+The project consists of an Amazon API Gateway, an AWS Lambda function, a DynamoDB Table, and an S3 bucket.  Although this project can be deployed, the focus of the code is to demonstrate local unit testing approaches using Python and mocking.  This project demonstrates testing locally an AWS Lambda function handler replacing remote API calls with mocks.  The Lambda function has optimizations for cold starts with global initializations, which are accomodated in the unit test.  
+
+---
+
+## Contents
+- [Test AWS Lambda function handlers locally using mocks (Python)](#test-aws-lambda-function-handlers-locally-using-mocks-python)
+  - [Introduction](#introduction)
+  - [Contents](#contents)
+  - [System Under Test (SUT)](#system-under-test-sut)
+  - [Goal](#goal)
+  - [Description](#description)
+  - [Sample Project](#sample-project)
+  - [Key Files in the Project](#key-files-in-the-project)
+  - [Prerequisites](#prerequisites)
+  - [Make commands for test and deployment](#make-commands-for-test-and-deployment)
+  - [Running the project](#running-the-project)
+  - [License](#license)
+---
+
+## System Under Test (SUT)
+
+The SUT in this pattern is a Lambda function that makes calls to other AWS cloud services using an AWS SDK.
+
+![System Under Test](/_img/pattern_04_lambda_mock_sut.png)
```

Comment:
Incorrect diagram? This is the same diagram as below. Tests and synthetic event are not part of the SUT.

Reply:
I triple-checked, and this is the diagram from our inventory.  Perhaps the inventory is also incorrect?

Reply:
Okay you are right. The inventory doc is wrong. However, this image does not accurately represent the architecture of the system under test in your sample. Could you create a new image from the PPT that does? 

Representations of tests should not appear in SUT diagrams.